### PR TITLE
improved toleration spec validation

### PR DESCRIPTION
**Pull request by jasmingacic**
# Summary of this proposal

improved toleration spec validation 

## Asana task(s) link.

https://app.shortcut.com/chronosphere/story/90187/validation-of-tolerations-parameter-in-the-cli

E2E tests https://github.com/chronosphereio/calyptia-cloud-e2e/pull/171

### CODE REVIEW

File: cmd/coreinstance/create_core_instance_operator.go [hunk 1]
```diff
@@ -276,7 +277,11 @@ func newCmdCreateCoreInstanceOperator(config *cfg.Config, testClientSet kubernet
 			}
 			syncDeployment, err := k8sClient.DeployCoreOperatorSync(ctx, coreCloudURL, coreDockerFromCloudImage, coreDockerToCloudImage, metricsPort, memoryLimit, annotations, tolerations, !noTLSVerify, httpProxy, httpsProxy, created, serviceAccount.Name)
 			if err != nil {
+				if err := config.Cloud.DeleteCoreInstance(ctx, created.ID); err != nil {
+					return fmt.Errorf("failed to rollback created core insatnce %v", err)
```
Comment (niedbalski) [inline cmd/coreinstance/create_core_instance_operator.go:281]: typo

File: k8s/client.go [hunk 2]
```diff
@@ -826,6 +855,8 @@ func ExtractGroupVersionResource(obj runtime.Object) (schema.GroupVersionResourc
 		Version:  gvk.Version,
 		Resource: gvk.Kind + "s",
 	}
+	fmt.Println("GVR", gvr)
```
Comment (niedbalski) [inline k8s/client.go:858]: I don't think we should be printing this here.

**Review by niedbalski — COMMENTED** [review]
@jasmingacic please cover on cloud-e2e

File: k8s/client.go (hunk 2 above)
Comment (jasmingacic) [reply k8s/client.go]: e2e is done here https://github.com/chronosphereio/calyptia-cloud-e2e/pull/171

**Review by lecaros — APPROVED** [review]
We reviewed this during a shared screen session with @jasmingacic. Please merge this.

**Review by niedbalski — APPROVED** [review]

**Pull request merged**